### ISSUE TITLE
Button: focus indicator does not meet color contrast on dark background

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1585]
> Anything that receives focus must have a 3:1 contrast between the focus indicator (visible when user TABs to the element) and its immediate background. Need to talk with Design Systems team on ensuring focus indicator ALWAYS meets contrast requirements. Temporary solution may be to lighten the background inside the button further to ensure 3:1 between button color and blue focus indicator. - Manasvi

![Screen Shot 2021-01-29 at 9 25 25 AM](https://user-images.githubusercontent.com/226356/106300684-08b6d580-6214-11eb-8550-a72c9cb4fc5c.png)

**Comment by jaketrent** [comment]
related: #1587

**Comment by sweeting** [comment]
Closed by #1593
